custom_components/moonraker/config_flow.py

```diff
@@ -18,7 +18,7 @@ def __init__(self):
         """Initialize."""
         _LOGGER.debug("loading moonraker confFlowHandler")
         self._errors = {}
-        self.title = None
+        self.title = DOMAIN
```

Comment:
I think we can just set it directly in async_create_entry

Reply:
Ideally I want this to be the hostname (can be next release)